projects/packages/jitm/src/class-jitm.php

```diff
@@ -36,7 +49,7 @@ public static function configure() {
 	 * @return Post_Connection_JITM|Pre_Connection_JITM JITM instance.
 	 */
 	public static function get_instance() {
-		if ( ( new Connection_Manager() )->is_connected() ) {
+		if ( ( new Connection_Manager() )->is_connected() || defined( 'IS_WPCOM' ) && IS_WPCOM ) {
```

Comment:
@scruffian You asked about whether the `IS_WPCOM` check should be part of `Connection_Manager`. The answer is I have no idea. It _could_ be. 

We'd need input from the Jetpack team on this I think.